ddtrace/contrib/internal/langchain/patch.py

```diff
@@ -905,10 +909,15 @@ def traced_similarity_search(langchain, pin, func, instance, args, kwargs):
         documents = func(*args, **kwargs)
         span.set_metric("langchain.response.document_count", len(documents))
         for idx, document in enumerate(documents):
+            if not isinstance(document, tuple):
```

Comment:
Do we re-use `document` elsewhere? If not, maybe we can do
```
for idx, document_and_maybe_score in enumerate(documents):
    if not isinstance(document_and_maybe_score, tuple):
        document, score = document_and_maybe_score, None
    else:
        document, score = document_and_maybe_score
    span.set_tag_str(
        "langchain.response.document.%d.page_content" % idx, integration.trunc(str(document.page_content))
    )
...
```
For me that is a bit more clear

Reply:
Hah I like the name ! Yes I don't believe it's used anywhere else.